### PR TITLE
Refer to nginx_route library instead of ingress

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ on:
 
 jobs:
   unit-tests:
-    uses: canonical/operator-workflows/.github/workflows/test.yaml@main
```
Comment (nrobinaubertin) [inline .github/workflows/test.yaml:8]: Do you want to merge this PR before the one on operator-workflows ?

Comment (arturo-seijas) [reply .github/workflows/test.yaml]: No, but well spotted

**Review by gtrkiller — DISMISSED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]

**Review by weiiwang01 — APPROVED** [review]

**Review by gtrkiller — APPROVED** [review]
LGTM

**Pull request merged**